src/index.js

```diff
@@ -153,12 +170,162 @@ const IN_CALL_BUTTONS_ARRAY = [
             }
         }
 
+        const renderAddNumber = () => {
+            const modalElement = document.createElement("div")
+            modalElement.id = "modal"
+            modalElement.style.visibility = "visible"
+
+            const modalControlWrapperElement = document.createElement("div")
+            modalControlWrapperElement.id = "modal-control-wrapper"
+
+            const cancelButtonElement = document.createElement("button")
+            cancelButtonElement.innerHTML = "Cancel"
+            cancelButtonElement.className = "modal-controls"
```

Comment:
Here, classname. Your should have global variable

`const CLASS_NAMES = {
  cancelButtonElement: 'modal-controls',
  ...
}`

and then use it in following way

`
cancelButtonElement.className = CLASS_NAMES.cancelButtonElement
`

It will reduce strings and prevent possible misstakes.